build.gradle

```diff
@@ -64,7 +64,8 @@ javadocJar.enabled = false
 ext {
     slf4jVersion = '1.7.7'
     servletApiVersion = '3.1.0'
-    zipkinVersion = '1.7.0'
+    zipkinCoreVersion = '1.7.0'
+    zipkinReporterVersion = '0.2.0'
```

Comment:
ps this api will thrash a while (pre 1.0), but it is unlikely people will have dependents on it in the same classpath.
choice is to accept the hazard or shade/shadow it until 1.0